install.sh

```diff
@@ -226,6 +243,7 @@ nvm_do_install() {
     fi
   fi
 
+  nvm_bash_completion
```

Comment:
Changing how the installer chooses a shell is a major, and separate PR :-)

If everything is consistent with `nvm_detect_profile`, then when it's changed later, everything will change along with it.